superset/config.py

```diff
@@ -695,3 +695,9 @@ class CeleryConfig(object):
     except Exception:
         logging.exception("Found but failed to import local superset_config")
         raise
+
+
+# Note currently SIP-15 feature is WIP and should not be enabled.
+SIP_15_ENABLED = False
+SIP_15_DEFAULT_TIME_RANGE_ENDPOINTS = ["unknown", "inclusive"]
+SIP_15_TOAST_MESSAGE = 'Preview then save your chart using the new time range endpoints <a href="{url}" class="alert-link">here</a>.'
```

Comment:
Maybe prefixing this with `Action Required:` or something like that to make it more apparent what's needed?

Reply:
@etr2460 one issue I ran into is that any more text would wrap in the toast which looked a tad weird. Do you know if these toasts can be resized?

Reply:
You could always increase the max size of the toasts. It seems like a reasonable change